debian/snapd.refresh.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=Automatically refresh installed snaps
-After=network.target snapd.socket
-Requisite=snapd.socket
+After=network-online.target snapd.socket
```

Comment:
probably, but it's not a regression: `network.target` would have the same trouble.